src/gui/selectivesyncwidget.cpp

```diff
@@ -263,8 +264,8 @@ void SelectiveSyncWidget::slotItemExpanded(QTreeWidgetItem *item)
     QString dir = item->data(0, Qt::UserRole).toString();
     if (dir.isEmpty())
         return;
-    // TODO: legacy
-    PropfindJob *job = new PropfindJob(_account, _account->davUrl(), _folderPath + dir, PropfindJob::Depth::One, this);
+    Q_ASSERT(!_davUrl.isEmpty());
```

Comment:
How about a getter with that check?

Reply:
Would save a tiny bit of code duplication, but we cannot enforce the use of that getter over the variable anyway...

Reply:
There are private getters :D
And in general I'd say we should always use getters instead of raw vars.

Reply:
Well, if you prefer it...

Reply:
Qt codestyle does not use get as getter prefix